libtsuba/src/LocalStorage.cpp

```diff
@@ -48,7 +48,10 @@ katana::LocalStorage::CleanUri(std::string* uri) {
   if (uri->find(uri_scheme()) != 0) {
     return;
   }
-  *uri = std::string(uri->begin() + uri_scheme().size(), uri->end());
+
+  std::string path(uri->begin() + uri_scheme().size(), uri->end());
```

Comment:
As an idiom to get a file name from a URI, this does not thrill me.

Reply:
I forgot to add this, but `CleanUri` can be replaced with `Uri::path`.